File: Problem Solving/Hard/half-done/array_manipulation/Python/array_manipulation_v1.py
def arrayManipulation(n, queries):
    result = [0] * n
    m = 0
    for i in range(len(queries)):
        a = queries[i][0] - 1
        b = queries[i][1] - 1
        c = queries[i][2]
        result = [result[_]+c if (_ >= a and _ <= b) else result[_]
                  for _ in range(len(result))]
        if(m < max(result)):
            m = max(result)
    return m

File: Problem Solving/Hard/half-done/array_manipulation/Javascript/array_manipulation_v1.js
function arrayManipulation(n, queries) {
    // Write your code here
    let result;
    let m = queries.length;
    let container = [[]];
    for (let i = 0; i < n; i++) {
        container[0].push(0);
    }

    for (let i = 0; i < m; i++) {
        let temp = [];
        let a = queries[i][0] - 1;
        let b = queries[i][1] - 1;
        let k = queries[i][2];

        for (let j = 0; j < n; j++) {
            if (j >= a && j <= b) {
                temp.push(container[i][j] + k);
            } else {
                temp.push(container[i][j]);
            }
        }

        container.push(temp);
    }
    container = [].concat(...container);
    result = Math.max(...container);

    return result;
}
